src/hotspot/share/gc/shenandoah/shenandoahScanRemembered.cpp

```diff
@@ -307,35 +313,39 @@ bool ShenandoahScanRemembered::is_write_card_dirty(size_t card_index) {
   return _rs->is_write_card_dirty(card_index);
 }
 
-bool ShenandoahScanRemembered::is_card_dirty(HeapWord *p) {
+bool ShenandoahScanRemembered::is_card_dirty(HeapWord* p) {
   return _rs->is_card_dirty(p);
 }
 
-void ShenandoahScanRemembered::mark_card_as_dirty(HeapWord *p) {
+void ShenandoahScanRemembered::mark_card_as_dirty(HeapWord* p) {
   _rs->mark_card_as_dirty(p);
 }
 
-void ShenandoahScanRemembered::mark_range_as_dirty(HeapWord *p, size_t num_heap_words) {
+bool ShenandoahScanRemembered::is_write_card_dirty(HeapWord* p) {
```

Comment:
Same question/remark as I asked/made for the ShenDirectCardMarkRS api above at line 91.

Reply:
Never mind; found where it's used now.